sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuColumnarToRowExec.scala

```diff
@@ -389,7 +389,9 @@ object GpuColumnarToRowExec {
     // Check if the current CUDA device architecture exceeds Pascal.
     // i.e. CUDA compute capability > 6.x.
     // Reference:  https://developer.nvidia.com/cuda-gpus
-    Cuda.getComputeCapabilityMajor > 6
+    //Cuda.getComputeCapabilityMajor > 6
+    // https://github.com/NVIDIA/spark-rapids/issues/10062, at least until we can debug and fix it.
+    false
```

Comment:
Normally I would  

1) create an internal conf (disabled by default).
2) parameterize the failing test 
3) xfail the failing `enabled` case

But it is ok either way as long as we are working on it